cmake/shared/PACE_Version.cmake

```diff
@@ -35,34 +35,39 @@ set(${PROJECT_NAME}_FULL_VERSION "${PROJECT_VERSION}")
 string(TOUPPER "${${PROJECT_NAME}_RELEASE_TYPE}" _release_type)
 
 if("${_release_type}" STREQUAL "NIGHTLY")
-    string(TIMESTAMP _date "%Y%m%d")
-    set(${PROJECT_NAME}_FULL_VERSION "${${PROJECT_NAME}_FULL_VERSION}-${_date}")
+  string(TIMESTAMP _date "%Y%m%d")
+  set(${PROJECT_NAME}_FULL_VERSION "${${PROJECT_NAME}_FULL_VERSION}-${_date}")
 endif()
 
 if(UNIX)
-    set(${PROJECT_NAME}_PLATFORM "linux")
+  set(${PROJECT_NAME}_PLATFORM "linux")
 elseif(WIN32)
-    set(${PROJECT_NAME}_PLATFORM "win64")
+  set(${PROJECT_NAME}_PLATFORM "win64")
 endif()
 
 set(${PROJECT_NAME}_FULL_VERSION "${${PROJECT_NAME}_FULL_VERSION}-${${PROJECT_NAME}_PLATFORM}-${Matlab_RELEASE}")
 
 if(NOT "${_release_type}" STREQUAL "RELEASE")
-    find_package(Git QUIET)
-    execute_process(
-        COMMAND ${GIT_EXECUTABLE} rev-list --abbrev-commit --no-merges -n 1 HEAD
-        WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
-        RESULT_VARIABLE _res
-        OUTPUT_VARIABLE GIT_REVISION_SHA
-        OUTPUT_STRIP_TRAILING_WHITESPACE
+  find_package(Git QUIET)
+  if ("${_release_type}" STREQUAL "NIGHTLY" OR "${_release_type}" STREQUAL "BRANCH")
```

Comment:
Would it be safer and clearer to make the skip use-case explicit as that is the exceptional rather than normal behaviour, i.e. reverse to logic
```
set(SKIP_MERGES = "") // or as an else
if (this-is-a-PR) 
  set(SKIP_MERGES = "--no-merge")
end
```